compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -1683,10 +1683,12 @@ class Objects:
     end if
 
     // class body
+    val whiteList = Set("scala.collection.immutable.Vector.emptyIterator")
```

Comment:
We should use a Definition here (from `dotty.tools.dotc.core.Definitions`) instead of a plain String, unless there is some reason not to.